Makefile

```diff
@@ -52,27 +52,28 @@ $(RUNSVINIT): vendor/runsvinit/*.go
 
 $(SCOPE_EXE): $(shell find ./ -path ./vendor -prune -o -type f -name *.go) prog/static.go $(CODECGEN_TARGETS)
 
-report/report.codecgen.go: $(call GET_CODECGEN_DEPS,report/)
+report/latest_map_impls_generated.go: report/latest_map_impls.go tools/generate_latest_map
+report/report.codecgen.go: $(call GET_CODECGEN_DEPS,report/) report/latest_map_impls_generated.go
```

Comment:
As discussed elsewhere, ditching the go generate step, so this shouldn't be an issue.